.github/workflows/test-php.yml

```diff
@@ -30,41 +29,69 @@ jobs:
         run: composer install --prefer-dist --no-progress --no-suggest
       - name: Run PHPCS
         run: composer run lint
+
   phpunit:
-      name: Phpunit
-      runs-on: ubuntu-latest
-      services:
-        mysql:
-          image: mysql:5.7
-          env:
-            MYSQL_ROOT_PASSWORD: root
-          ports:
-            - 3306/tcp
-          options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
-      steps:
-        - name: Setup PHP version
-          uses: shivammathur/setup-php@v1
-          with:
-            php-version: '7.2'
-            extensions: simplexml, mysql
-            tools: phpunit:7.5.15
-        - name: Checkout source code
-          uses: actions/checkout@v2
-        - name: Install WordPress Test Suite
-          run: |
-            bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1:${{ job.services.mysql.ports['3306'] }}
-        - name: Get Composer Cache Directory
-          id: composer-cache
-          run: |
-            echo "::set-output name=dir::$(composer config cache-files-dir)"
-        - name: Setup Composer cache
-          uses: actions/cache@v1
-          with:
-            path: ${{ steps.composer-cache.outputs.dir }}
-            key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-            restore-keys: |
-              ${{ runner.os }}-composer-
-        - name: Install composer
-          run: composer install --prefer-dist --no-progress --no-suggest
-        - name: Run phpunit
-          run: phpunit
+    name: Phpunit
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306/tcp
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - name: Setup PHP version
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.2'
```

Comment:
Can this also be 7.4 like in PHPStan? Or maybe 8.X?

Reply:
I think I need to change these and bring them down to Otter's Minimum PHP support instead of taking it higher than the minimum support we offer. We currently have 5.4 but we plan on dumbing it to 5.6 so it should ideally be on 5.6 to make sure our plugin is compatible with what we claim. What do you think?

Reply:
For running tests, is ok 5.6. But for checking the code style or formatting of the file, it should be 8.X since it is a dev tool.

Reply:
Actually, the unit test should be with 5.6. E2E should be also 8.X since they are decoupled from the implementation.

Reply:
E2E aren't on 8.x because I remember last time I tried, PHPUnit wasn't compatible with 8. I think we can try again now, a new version of PHPUnit was released in Feb which is compatible with PHP 8.1.